torch/csrc/jit/mobile/import.cpp

```diff
@@ -226,6 +252,113 @@ void parseMethods(
   }
 }
 
+// Find if any of the methods in the bytecode model bytecode_values include
+// a tuple with this format ('tensor_jit_index', 4) in "constants" field.
+bool has_tensor_jit_index(const std::vector<IValue>& bytecode_values) {
+  // The following variables are used to locate the "constants"
+  // fields in elements
```

Comment:
I love the comments you have in here! I'm wondering if you could split the implementation into helper methods too for ease of understanding?